Security check fail with "padraic/humbug_get_contents" old version 1.0.4

Hello,
I have noticed an issue recently :
The package "padraic/phar-updater" requires "padraic/humbug_get_contents" version 1.0.4 but not newest version 1.1.2, which create failure in security check.

Are you going to update package "padraic/phar-updater" for solving this issue ?

Thanks in advance.

----------------------------------------------

Symfony Security Check Report
=============================

 // Checked file: /my_project/apache/volume/composer.lock

 [ERROR] 1 packages have known vulnerabilities.                                 

padraic/humbug_get_contents (1.0.4)
-----------------------------------

 * CVE-2016-5385: HTTP Proxy header vulnerability
   https://github.com/humbug/file_get_contents/releases/tag/1.1.2

 ! [NOTE] This checker can only detect vulnerabilities that are referenced in   
 !        the SensioLabs security advisories database. Execute this command     
 !        regularly to check the newly discovered vulnerabilities.              

Loaded config default from ".php_cs.dist".

--- a/composer.json
+++ b/composer.json
@@ -13,7 +13,7 @@
     ],
     "require": {
         "php": ">=5.3.3",
-        "padraic/humbug_get_contents": "1.0.4"
+        "padraic/humbug_get_contents": "^1.0"
     },
     "require-dev": {
         "phpunit/phpunit": "~4.0"

--- a/src/Updater.php
+++ b/src/Updater.php
@@ -301,11 +301,19 @@ class Updater
 
     public function throwRuntimeException($errno, $errstr)
     {
+        if (E_USER_DEPRECATED === $errno) {
+            return;
+        }
+
         throw new RuntimeException($errstr);
     }
 
     public function throwHttpRequestException($errno, $errstr)
     {
+        if (E_USER_DEPRECATED === $errno) {
+            return;
+        }
+
         throw new HttpRequestException($errstr);
     }
 

